### ISSUE TITLE
Add dependency on libwebkit2gtk-4.0-37-gtk2

### ISSUE THREAD

**Issue by fenryxo** [tiliado/nuvolaplayer#232]
**This issue has been fixed in the [master](https://github.com/tiliado/nuvolaplayer) & [3.0.x](https://github.com/tiliado/nuvolaplayer/tree/3.0.x) GIT branches. Thanks for your bug report.**

At first, this fix will be available with the next rebuilt of packages of the [Nuvola Player 3.1 Rolling Releases](https://tiliado.github.io/nuvolaplayer/documentation/3.1/install.html). Later, the fix will be also included in the next bug-fix release of the [Nuvola Player 3.0 Stable Releases](https://tiliado.github.io/nuvolaplayer/documentation/3.0/install.html). To get informed when Nuvola Player 3.0.3 Stable Release is released,

  * follow Nuvola Player on [Facebook](https://www.facebook.com/nuvolaplayer),
    [Google+](https://plus.google.com/110794636546911932554) or
    [Twitter](https://twitter.com/NuvolaPlayer).
  * subscribe to the Nuvola Player Newsletter: [weekly](http://eepurl.com/bLbm5H)
    or [monthly](http://eepurl.com/bLbtM1).

---

WebKitPluginProcess2 was moved to a libwebkit2gtk-4.0-37-gtk2 separate package since 2.10.4+dfsg1-1.

```
webkit2gtk (2.10.4+dfsg1-1) unstable; urgency=medium

  * New upstream release.
  * debian/patches/install-minibrowser.patch:
    + Refresh.
  * webkit2gtk includes a binary called WebKitPluginProcess2 that allows
    loading plugins that depend on GTK+2, like Adobe Flash or Google
    Hangouts. This however has the side effect of requiring GTK+2 and all
    its dependencies. We're moving the WebKitPluginProcess2 to a separate
    package so users that don't need any of those plugins don't need to
    install GTK+2 (Closes: #804412).
    + debian/control:
      - Add the entry for the new libwebkit2gtk-4.0-37-gtk2 package.
    + debian/rules:
      - Run dh_strip on the new package as well.
    + debian/patches/hide-gtk2-plugins.patch:
      - Hide plugins if the plugin process is not available.
    + debian/libwebkit2gtk-4.0-37-gtk2.install:
      - Install WebKitPluginProcess2.
    + debian/libwebkit2gtk-4.0-37.install:
      - Remove WebKitPluginProcess2.
    + debian/NEWS:
      - Add news item about the new package.

 -- Alberto Garcia <berto@igalia.com>  Thu, 12 Nov 2015 23:10:53 +0200
```

**Comment by fenryxo** [comment]
Ubuntu has libwebkit2gtk-4.0-37-gtk2 2.10.9-1ubuntu1 in xenial and newer.

**Comment by fenryxo** [comment]
Add new dependency `libwebkit2gtk-4.0-37-gtk2 | libwebkit2gtk-4.0-37 (<< 2.10.4+dfsg1-1)`

**Comment by fenryxo** [comment]
**This issue has been fixed in the [master](https://github.com/tiliado/nuvolaplayer) & [3.0.x](https://github.com/tiliado/nuvolaplayer/tree/3.0.x) GIT branches. Thanks for your bug report.**

At first, this fix will be available with the next rebuilt of packages of the [Nuvola Player 3.1 Rolling Releases](https://tiliado.github.io/nuvolaplayer/documentation/3.1/install.html). Later, the fix will be also included in the next bug-fix release of the [Nuvola Player 3.0 Stable Releases](https://tiliado.github.io/nuvolaplayer/documentation/3.0/install.html). To get informed when Nuvola Player 3.0.3 Stable Release is released,

  * follow Nuvola Player on [Facebook](https://www.facebook.com/nuvolaplayer),
    [Google+](https://plus.google.com/110794636546911932554) or
    [Twitter](https://twitter.com/NuvolaPlayer).
  * subscribe to the Nuvola Player Newsletter: [weekly](http://eepurl.com/bLbm5H)
    or [monthly](http://eepurl.com/bLbtM1).

**Comment by fenryxo** [comment]
The fix has been released in Nuvola Player 3.0.3. Thanks for your bug report.
